tests/components/tuya/fixtures/cs_smart_dry_plus.json

```diff
@@ -0,0 +1,32 @@
+{
+  "endpoint": "https://apigw.tuyaeu.com",
+  "terminal_id": "mock_terminal_id",
+  "mqtt_connected": true,
+  "disabled_by": null,
+  "disabled_polling": false,
+  "id": "mock_device_id",
+  "name": "Dehumidifier ",
+  "category": "cs",
+  "product_id": "vmxuxszzjwp5smli",
+  "product_name": "the Smart Dry Plus\u2122 Connect Dehumidifier ",
+  "online": true,
+  "sub": false,
+  "time_zone": "+10:00",
+  "active_time": "2024-05-28T01:57:58+00:00",
+  "create_time": "2024-05-28T01:57:58+00:00",
+  "update_time": "2024-05-28T01:57:58+00:00",
+  "function": {},
+  "status_range": {
+    "fault": {
+      "type": "Bitmap",
+      "value": {
+        "label": ["E1", "E2"]
+      }
+    }
+  },
+  "status": {
+    "fault": 0
+  },
```

Comment:
`function` is empty for this dehumidifier.
We probably shouldn't be creating `FAN` or `HUMIDIFIER` entities for it - but that can be done in a follow-up PR.